### PR TITLE
Should send the OCI runtime path not just the name to buildah

### PR DESCRIPTION
**Pull request by rhatdan**
[NO TESTS NEEDED] Mainly because I have no idea how we would test
this.

Fixes: https://github.com/containers/podman/issues/9459

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: libpod/runtime_img.go [hunk 1]
```diff
@@ -168,9 +168,7 @@ func (r *Runtime) newImageBuildCompleteEvent(idOrName string) {
 // Build adds the runtime to the imagebuildah call
 func (r *Runtime) Build(ctx context.Context, options buildahDefine.BuildOptions, dockerfiles ...string) (string, reference.Canonical, error) {
 	if options.Runtime == "" {
-		// Make sure that build containers use the same runtime as Podman (see #9365).
-		conf := util.DefaultContainerConfig()
-		options.Runtime = conf.Engine.OCIRuntime
+		options.Runtime = r.GetOCIRuntimePath()
```
Comment (TomSweeneyRedHat) [inline libpod/runtime_img.go:171]: Could the default runtime ever differ from what's set in the containerConfig structure?  If so, I'm not sure we'd want to do this.

**Comment by TomSweeneyRedHat** [comment]
One question, if I'm off base with it, then LGTM.

File: libpod/runtime_img.go (hunk 1 above)
Comment (rhatdan) [reply libpod/runtime_img.go:171]: They are stored in the same structure.  We read it originally from the containers.conf, but if the user overrides it, we replace the value in the same structure.

**Pull request merged**